packages/@aws-cdk/aws-ecs/test/container-definition.test.ts

```diff
@@ -2033,4 +2033,30 @@ describe('container definition', () => {
     });
 
   });
+
+  test('exposes image name', () => {
+    // GIVEN
+    const stack = new cdk.Stack();
+    const taskDefinition = new ecs.FargateTaskDefinition(stack, 'TaskDef');
+
+    // WHEN
+    const container = taskDefinition.addContainer('cont', {
+      image: ecs.ContainerImage.fromAsset(path.join(__dirname, 'demo-image')),
+    });
+
+    // THEN
+    expect(stack.resolve(container.imageName)).toEqual({
```

Comment:
The unit tests for this module have been migrated to Assertions since this PR was opened! Please merge the latest from master, and update this unit test. (Should be pretty minimal effort).

Reply:
There's nothing to update here, it's an expectation on `stack.resolve()`.

Reply:
@madeline-k?